File: Haskell/start-code/startcode.hs
-- Willkommen zu meinem Haskell-Programm

main :: IO ()
main = do
    -- Begrüßung
    putStrLn "Willkommen zu meinem Haskell-Programm!"

    -- Nutzereingabe
    putStr "Bitte geben Sie Ihren Namen ein: "
    name <- getLine

    -- Begrüßung mit dem Namen
    putStrLn ("Hallo, " ++ name ++ "! Schön, dass Sie hier sind.")
    putStrLn "Das Programm wurde beendet."

File: Java/start-code/startcode.java
// Beschreibung: Ein einfaches Java-Programm
// Datum: 25.01.2025

import java.util.Scanner;

public class StartCode {
    public static void main(String[] args) {
        // Begrüßung
        System.out.println("Willkommen zu meinem Java-Programm!");

        // Scanner für Nutzereingabe initialisieren
        Scanner scanner = new Scanner(System.in);

        // Nutzereingabe abfragen
        System.out.print("Bitte geben Sie Ihren Namen ein: ");
        String name = scanner.nextLine();

        // Begrüßung mit dem Namen
        System.out.println("Hallo, " + name + "! Schön, dass Sie hier sind.");

        // Scanner schließen
        scanner.close();

        // Programm beenden
        System.out.println("Das Programm wurde beendet.");
    }
}
